docs/section1/Rule1-1.md

```diff
@@ -0,0 +1,43 @@
+# Section 1 - Rule 1-1
+**Schema Version:** 0.0.29  
+**Mandatory Rule:** True  
+**Rule ID:** 1-1
+**Rule Description:** Building performance factors shall be from Standard 90.1-2019, Table 4.2.1.1, based on the building area type and climate zone. For building area types not listed in Table 4.2.1.1  “All others.” shall be used to determine the BPF.
+**Rule Assertion:** Options are PASS/FAIL/UNDETERMINED     
+**Appendix G Section:** Section 4.2.1.1  
+**90.1 Section Reference:** None  
+
+**Data Lookup:** Table 4.2.1.1 
+
+**Evaluation Context:** Output2019ASHRAE901
+
+**Applicability Checks:** None
+
+**Function Calls:**
+get_BPF_building_area_types_and_zones()
+
+## Rule Logic:
+- Get the dictionary of interpreted BPF building area types for the project: `bpf_building_area_type_dict = get_BPF_building_area_types_and_zones(B_RMD)`
+- If the BPF building area type dictionary contains any building segments with UNDETERMINED BPF building area type; outcome = UNDETERMINED: `if "UNDETERMINED" in bpf_building_area_type_dict: UNDETERMINED and raise_message "One or more building area types could not be determined for the project's building segments."`
+- Get the project climate zone: `climate_zone = RulesetProjectDescription.weather.climate_zone`
+- Create a variable to store the summed product of BPF and Area: `bpf_bat_sum_prod = 0`
+- Create a variable to store the expected summed product of BPF and Area: `expected_bpf_bat_sum_prod = 0`
+- Create a variable to store the Total Area: `total_area = 0`
+- Iterate through the building area type(s) in the BPF building area type dictionary: `for bpf_bat in bpf_building_area_type_dict:`
+  - Get the expected BPF value from Table 4.2.1.1 based on building area and climate zone: `expected_bpf = data_lookup(Table_4_2_1_1, bpf_bat, climate_zone)`
+  - Get the output BPF value: `output_bpf = Output2019ASHRAE901.total_area_weighted_building_performance_factor`
+  - Add the area to the total: `total_area += bpf_building_area_type_dict[bpf_bat]["AREA"]`
+  - Add the product of expected BPF and area to the expected summed product of BPF and area: `expected_bpf_bat_sum_prod += (expected_bpf * bpf_building_area_type_dict[bpf_bat]["AREA"])`
+  - Add the product of the output BPF and area to the summed product of BPF and area: `bpf_bat_sum_prod += (output_bpf * bpf_building_area_type_dict[bpf_bat]["AREA"])`
```

Comment:
Is this output_bpf  by building segment or across the entire RMD? Just asking because not sure if it makes sense to multiply by the area if it is not building segment specific.

Reply:
Good catch, I made a mistake here.